source/gui/addonStoreGui/controls/messageDialogs.py

```diff
@@ -578,9 +578,9 @@ def handleDisplayableError(displayableError: DisplayableError):
 	@classmethod
 	def _checkForUpdatableAddons(cls):
 		if not NVDAState.shouldWriteToDisk() or (
-			AddonsAutomaticUpdate.DISABLED == config.conf["addonStore"]["automaticUpdates"]
+			AddonsUpdateMode.DISABLED == config.conf["addonStore"]["automaticUpdates"]
 		):
-			log.debug("automatic add-on updates are disabled")
+			log.debug("add-on updates are disabled")
```

Comment:
I think this is a bit misleading - as it doesn't affect the update behaviour from the update panel in the add-on store